.github/workflows/autocorrect.yml

```diff
@@ -0,0 +1,14 @@
+name: AutoCorrect
```

Comment:
Personally, I'd rather do it manually from time to time. Not sure about others opinions.